sqlalchemy-stubs/dialects/mssql/__init__.pyi

```diff
@@ -1,38 +1,33 @@
 from .base import (
+    INTEGER as INTEGER,
     BIGINT as BIGINT,
-    BINARY as BINARY,
-    BIT as BIT,
-    BLOB as BLOB,
-    BOOLEAN as BOOLEAN,
+    SMALLINT as SMALLINT,
+    TINYINT as TINYINT,
+    VARCHAR as VARCHAR,
+    NVARCHAR as NVARCHAR,
     CHAR as CHAR,
-    DATE as DATE,
-    DATETIME as DATETIME,
-    DECIMAL as DECIMAL,
-    DOUBLE as DOUBLE,
-    ENUM as ENUM,
-    DECIMAL as DECIMAL,
-    FLOAT as FLOAT,
-    INTEGER as INTEGER,
-    INTEGER as INTEGER,
-    JSON as JSON,
-    LONGBLOB as LONGBLOB,
-    LONGTEXT as LONGTEXT,
-    MEDIUMBLOB as MEDIUMBLOB,
-    MEDIUMINT as MEDIUMINT,
-    MEDIUMTEXT as MEDIUMTEXT,
     NCHAR as NCHAR,
-    NVARCHAR as NVARCHAR,
-    NUMERIC as NUMERIC,
-    SET as SET,
-    SMALLINT as SMALLINT,
-    REAL as REAL,
     TEXT as TEXT,
+    NTEXT as NTEXT,
+    DECIMAL as DECIMAL,
+    NUMERIC as NUMERIC,
+    FLOAT as FLOAT,
+    DATETIME as DATETIME,
+    DATETIME2 as DATETIME2,
+    DATETIMEOFFSET as DATETIMEOFFSET,
+    DATE as DATE,
     TIME as TIME,
-    TIMESTAMP as TIMESTAMP,
-    TINYBLOB as TINYBLOB,
-    TINYINT as TINYINT,
-    TINYTEXT as TINYTEXT,
+    SMALLDATETIME as SMALLDATETIME,
+    BINARY as BINARY,
     VARBINARY as VARBINARY,
-    VARCHAR as VARCHAR,
-    YEAR as YEAR,
+    BIT as BIT,
+    REAL as REAL,
+    IMAGE as IMAGE,
+    TIMESTAMP as TIMESTAMP,
+    ROWVERSION as ROWVERSION,
+    MONEY as MONEY,
+    SMALLMONEY as SMALLMONEY,
+    UNIQUEIDENTIFIER as UNIQUEIDENTIFIER,
+    SQL_VARIANT as SQL_VARIANT,
+    XML as XML
```

Comment:
It is always good to minimize the diff to simplify comparison. For example here it looks like imports are sorted differently in your branch.

Reply:
It is actually easier to review. For example I can easily spot whether something is deleted.  (Sorry if this sounds a bit paranoid.)